Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3_test.py

```diff
@@ -1404,3 +1409,169 @@ def test_update_status(mocker):
     assert result.outputs["code"] == 204
     assert isinstance(result.outputs["message"], str)
     assert result.outputs["Workbench_Id"] == "WB-20837-20220418-00000"
+
+
+# Mock function for run custom script
+def run_custom_script_mock_response(*args, **kwargs):
```

Comment:
As mentioned above, you can find a good example for given-when-then doc strings in crowdstrikeflacon_test.py file.